e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -344,6 +344,100 @@ test('Example Imagery in Display layout', async ({ page }) => {
     expect(backgroundImageUrl2 >= backgroundImageUrl1);
 });
 
+test.describe('Example imagery thumbnails resize in display layouts', () => {
+
+    test('Resizing the layout changes thumbnail visibility and size', async ({ page }) => {
+        await page.goto('/', { waitUntil: 'networkidle' });
+
+        const thumbsWrapperLocator = await page.locator('.c-imagery__thumbs-wrapper');
+        // Click button:has-text("Create")
+        await page.locator('button:has-text("Create")').click();
+
+        // Click li:has-text("Display Layout")
+        await page.locator('li:has-text("Display Layout")').click();
+
+        // Double click text=Properties Title Notes Horizontal grid (px) Vertical grid (px) Horizontal size ( >> input[type="text"]
+        await page.locator('text=Properties Title Notes Horizontal grid (px) Vertical grid (px) Horizontal size ( >> input[type="text"]').dblclick();
```

Comment:
which element is this?

Reply:
This is selecting the domain objects text field during object creation mode to set it to 'Thumbnail Display Layout' on line 366.

Reply:
@michaelrogers ok in THIS case, I think we should to a locator replacement CONST to make it clear which component is being interacted with